test/integration/api.bats

```diff
@@ -9,7 +9,7 @@ function teardown() {
 
 @test "docker info should return the number of nodes" {
 	start_docker 3
-	start_manager
+	swarm_manage `join , ${HOSTS[@]}`
```

Comment:
Awesome.

I think it could be something like: You can optionally pass a parameter to `swarm_manage`, in which case it'll use it as the address you want to manage (such as `swarm manage etcd://...`).

By default, if no parameters are passed, it will default to `join , ${HOSTS[@]}`.

I think it's a good tradeoff since only discovery tests will actually want to pass a custom discovery address, all the other tests will be happy with the default one and it'll make them simpler.